loops.js

```diff
@@ -10,10 +10,11 @@
 // Tip: je kunt de .repeat() methode gebruiken om een karakter een n aantal keer te herhalen...
 // Dit heb je nog niet geleerd, maar bekijk hiervoor dit MDN-artikel eens: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat
 // ==========================================
-
-
-
-
+for (let i = 1; i <= 5; i++) { //Deze loop loopt van 1 t/m 5, wat overeenkomst met het aantal rijen in het patroon.
+    let row =''; //Dit initialiseert een lege String die we zullen vullen met de juiste asterisken.
+    for (let j = 1; j <= i; j++) { row += '*'; } //Deze innerlijke for loop voegt de juiste hoeveelheid asterisken toe aan de rij, gebaseerd op de huidige waarde van 'i'.
```

Comment:
Een loop in een loop was hier niet persé nodig. Je had hier namelijk de repeat methode kunnen gebruiken.

for (let i = 0; i < 5; i++) {
  console.log('*'.repeat(i + 1));
}